README.md

```diff
@@ -562,7 +562,7 @@ The name of the PuppetDB service. Defaults to `puppetdb`.
 
 #### `puppetdb_service_status`
 
-Sets whether the service should be `running ` or `stopped`. When set to `stopped` the
+Sets whether the service should be `running` or `stopped`. When set to `stopped` the
```

Comment:
nitpick: such README.md cleanups could also be submitted in a dedicated PR if we aim for perfect/clean PRs. But I think it's also fine to keep the changes here.

Reply:
Yeah, that was a byproduct of my editor's auto-cleanup tool. :facepalm: